### DISCUSSION TITLE
How to render a project file as if it were a single qmd file?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#4029] [Q&A] [answered]
Hi,
I have a quarto website project, in which in example I have the top and the side menu.

Is there a way to render in HTML a single qmd file I have in this project, as if it were a single qmd file, and in example not to have all the menu items I have in the side and without the top bar?

Thank you

**Comment by dragonstyle** [comment, marked as answer]
You can suppress the sidebar / navbar on a page using:

```yaml
sidebar: false
navbar: false
```

Would that work?

**Reply by aborruso** [reply]
Yes, thank you

**Reply by aborruso** [reply]
@dragonstyle  Can I add these in some way also via cli, without modifying the source file?

**Reply by dragonstyle** [reply]
You can pass them via a metadata file. I made:

**no-nav.yml**
```yaml
sidebar: false
navbar: false
```

and then used:

```bash
$ quarto render test.qmd --metadata-file no-nav.yml
```

This should do it without needing to actually modify the file!

**Reply by aborruso** [reply]
great thank you again @dragonstyle
